content/cadre.hbs

```diff
@@ -34,17 +34,7 @@
 						</div>
 					</div>
 				</div>
-				<div class="column is-one-third">
-					<div class="card">
-						<div class="card-image">
-							<figure class="babcock"></figure>
-						</div>
-						<div class="card-content">
-							<h1>Captain Babcock</h1>

```

Comment:
The README should probably be updated to reflect Capt Babcock no longer being a point-of-contact